priam/src/main/java/com/netflix/priam/defaultimpl/PriamGuiceModule.java

```diff
@@ -86,10 +95,38 @@ protected void configure() {
 
     @Provides
     @Singleton
-    Client provideJerseyClient() {
-        return new JerseyClientBuilder()
+    Client provideJerseyClient(MetricRegistry metricRegistry) {
+        return new JerseyClientBuilder(metricRegistry)
                 .using(priamConfiguration.getHttpClientConfiguration())
                 .using(environment)
-                .build();
+                .build("priam");
+    }
+
+    @Provides
+    @Singleton
+    @Port
```

Comment:
Instead of creating and binding a `Port` annotation to an Integer it may be cleaner to bind a `com.google.common.net.HostAndPort` using the bindHost and port from the `HttpConnectorFactory`.